src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -470,13 +472,12 @@ private static bool MatchAttributeTarget(IAttributeTargetSymbol attributeTarget,
                 {
                     //NOTE: ValueText so that we accept targets like "@return", to match dev10 (DevDiv #2591).
                     diagnostics.Add(ErrorCode.WRN_InvalidAttributeLocation,
-                        targetOpt.Identifier.GetLocation(), targetOpt.Identifier.ValueText);
+                        targetOpt.Identifier.GetLocation(), targetOpt.Identifier.ValueText, allowedTargets.ToDisplayString());
```

Comment:
Attribute targets are language keywords, I think they shouldn't be localized in context of this message because the message suggests what you can type here.